tensorflow/BUILD

```diff
@@ -89,6 +89,12 @@ config_setting(
 )
 
 config_setting(
+    name = "linux_ppc64le",
```

Comment:
Can you make this into a separate rule that also checks for jemalloc?
"with_jemalloc_ppc64le"